### DISCUSSION TITLE
App references non-public selectors in Payload ....App Store Connect warning

### DISCUSSION THREAD

**Discussion by ChandraKAV** [ionic-team/capacitor#5931] [General] [answered]
This warning does not allow upload of the app to App Store Connect. Not sure where the problem is ..I am using both Cordova and Capacitor plugins in this app ..

**Warning**: App references non-public selectors in Payload/xxxxxx.app/xxxxxx: transform: (no more detail) ....

I am stuck with this ...please help.

**Ionic Info**:

   Ionic CLI                     : 6.20.1 (/Users/iosdev/.nvm/versions/node/v16.17.0/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 6.2.6
   @angular-devkit/build-angular : 14.2.2
   @angular-devkit/schematics    : 14.2.2
   @angular/cli                  : 14.2.2
   @ionic/angular-toolkit        : 6.1.0

Cordova:

   Cordova CLI       : 11.0.0
   Cordova Platforms : ios 6.2.0
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 5.0.0, (and 20 other plugins)

Utility:

   cordova-res : 0.15.4
   native-run  : 1.7.0

System:

   ios-deploy : 1.11.4
   ios-sim    : 8.0.2
   NodeJS     : v16.17.0 (/Users/iosdev/.nvm/versions/node/v16.17.0/bin/node)
   npm        : 8.15.0
   OS         : macOS Monterey
   Xcode      : Xcode 14.0 Build version 14A309

iosdev@iosdevAir ccpro-mobile %

**Comment by jcesarmobile** [comment, marked as answer]
Do you have some Firebase plugin installed?
I saw an user with a similar problem and the issue was in their Firebase version.

**Reply by ChandraKAV** [reply]
Yes. I do have Firebase Messaging .. "cordova-plugin-fcm-with-dependecy-updated": "^7.8.0" .. I will check by removing this plugin ... Thanks for the info ..

**Reply by ChandraKAV** [reply]
Yes. That worked. Many thanks.
